types/applications/application.d.ts

```diff
@@ -428,13 +428,13 @@ declare namespace Application {
     /**
      * @defaultValue `[]`
      */
-    dragDrop: DragDrop[];
+    dragDrop: DragDrop.Options[];
 
     /**
      * Track Tab navigation handlers which are active for this Application
      * @defaultValue `[]`
      */
-    tabs: Omit<Tabs.Options, 'callback'>[];
```

Comment:
Is there a reason you changed this? When creating an application, `callback` will be overridden, so it doesn't really make sense to allow it in the `Application.Options`. People would only think that they can provide a callback while actually it's completely ignored (see `foundry.js#4331`). The same is true for the `DragDrop.Options` (`foundry.js#4309`, `permissions` also seems to be overridden here) and `SearchFilter.Options` (`foundry.js#4345`).